src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -222,7 +222,7 @@ mkImage options (T.unpack -> src) = do
    let attr = ("",[], kvs)
    let alt = str "image"
    defaultExt <- getOption readerDefaultImageExtension
-   let exts' = [".pdf", ".png", ".jpg", ".mps", ".jpeg", ".jbig2", ".jb2"]
+   let exts' = [".pdf", ".png", ".jpg", ".mps", ".jpeg", ".jbig2", ".jb2", ".svg"]
```

Comment:
I think this change could be problematic, as it would modify the behavior of `\includegraphics`. Maybe the list of default extensions could be passed in as a parameter?